modules/core/src/webgl/classes/renderbuffer.js

```diff
@@ -90,4 +108,38 @@ export default class Renderbuffer extends Resource {
     // this.gl.bindRenderbuffer(GL.RENDERBUFFER, null);
     return value;
   }
+
+  /* eslint-disable complexity */
+  _getFormatSize() {
```

Comment:
Idea: We already have a table for formats, maybe reuse that or add this field to that table?

See top of file:
```
import RENDERBUFFER_FORMATS from './renderbuffer-formats';
```

Reply:
Good call. And since the formats are the same between textures and renderbuffers, I could have one for both.